Gemfile

```diff
@@ -117,7 +117,7 @@ gem "canonical-rails"
 gem "logstasher"
 
 # Used to generate images for traces
-gem "gd2-ffij", :git => 'git@github.com:mmd-osm/gd2-ffij.git', :branch => 'animated_gif'
+gem "gd2-ffij", :git => "git@github.com:mmd-osm/gd2-ffij.git", :branch => "animated_gif"
```

Comment:
The git protocol link causes some issues with TravisCI. Maybe try `https://github.com/mmd-osm/gd2-ffij.git` instead?